### ISSUE TITLE
Leadership Network: Team Kick-Off

### ISSUE THREAD

**Issue by ldecoursy** [MozillaFoundation/MoFoLondon#26]
## Description:

Gather all MLN staff at the beginning of the week for team-building, reviewing the arc of the week, expectations, etc. 
### Time requirements:

1-2 hours
### Space requirements:

Need large space for 50-ish people
### Required attendees:

MLN staff & volunteers
### Dependencies: (Other sessions that need to happen before this one)

Should happen Tuesday end of day or first thing Wednesday

**Comment by ldecoursy** [comment]
May just do this via a staff call the week before instead.

**Comment by OpenMatt** [comment]
@ldecoursy sounds like this is no longer needed right?
